CSP/find_domain.py

```diff
@@ -0,0 +1,56 @@
+# This file contains the function find_domain() which will generate the starting domain for one note of a given line
+# with the harmony given as a parameter for context. Also implemented here is the helper function expand_harmony().
+
+import itertools
+
+
+# This function takes a parameter harmony and returns the pitch content of that chord
+# Parameter harmony is a list of the format [root, quality, inversion]
+def expand_harmony(harmony):
+    pitch_content = []
+
+    # Adding the root
+    if harmony[0] == 'A':
+        pitch_content.append(10)
+    elif harmony[0] == 'B':
+        pitch_content.append(11)
+    else:
+        pitch_content.append(harmony[0])
+
+    root = pitch_content[0]
+
+    # Adding the 3rd
+    if harmony[1] in ['M', '7']:
+        pitch_content.append((root + 4) % 12)
+    elif harmony[1] in ['m', 'b', 'D', 'd']:
+        pitch_content.append((root + 3) % 12)
```

Comment:
We should use `else` rather than `elif` right?  The difference between this and the fifth makes me worry that the 3rd is implied not always present.